app/javascript/shared/components/LabwareScan.vue

```diff
@@ -184,12 +179,8 @@ export default {
       }
     },
     computedValidators() {
-      if (this.validators) {
-        return this.validators
-      }
-
       if (this.labwareType == 'tube') {
-        return []
+        return [checkState(['passed'])]
```

Comment:
So this is changing the behaviour for everywhere that this component is used - do we think that's OK?

Reply:
I'm not sure if it's safe. I'd have to look at where else it's being used and even then I'm not sure I could say whether we can assume it's always passed or not. I think it's quite likely that tag plates are scanned in in a 'pending' state... but then, that's a plate not a tube. Possibly the only other place it's being used is in Cardinal, in which case this would be fine. 
I guess the alternative would be to pass the validation rule in as a prop (the thing you got rid of)?

Reply:
Yep I think it's fine to add this for Cardinal too, because they're doing the same thing in the lab in both pipelines.